### PR TITLE
Add local dev fix for prompts without reviewers

### PR DESCRIPTION
**Pull request by ericliuche**
## Description
Applies default settings to prompts without users

## Test plan
Ran locally

### CODE REVIEW

File: pkg/server/apiext/apiext.go [hunk 1]
```diff
@@ -457,6 +457,21 @@ func GetPromptHandler(ctx context.Context, state *state.State, r *http.Request)
 
 	for _, p := range run.Prompts {
 		if p.ID == promptID {
+			reviewersWithDefaults := p.Reviewers
```
Comment (colinking) [inline pkg/server/apiext/apiext.go]: You'll need to fix `ListPrompts` too

Comment (colinking) [reply pkg/server/apiext/apiext.go]: Also we should fix `web` so it works with older CLIs.

**Comment by ericliuche** [comment]
ah one sec, gotta fix /prompts/list

File: pkg/server/apiext/apiext.go (hunk 1 above)
Comment (colinking) [inline pkg/server/apiext/apiext.go]: Do you need a separate variable? Can you just set `p.Reviewers` directly?

**Review by colinking — APPROVED** [review]
nice

**Pull request merged**